content/v2/domains/dnssec.markdown

```diff
@@ -0,0 +1,238 @@
+---
+title: DNSSEC API | Domains | DNSimple API v2
+excerpt: This page documents the DNSimple DNSSEC API v2.
+---
+
+# DNSSEC API
+
+<note>
+This article describes a feature in Public Beta.
+</note>
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+
+## Enable DNSSEC {#enable}
+
+    POST /:account/domains/:domain/dnssec
+
+Enable DNSSEC for the domain in the account. This will sign the zone. If the domain is registered it will also add the DS record to the corresponding registry.
+
```

Comment:
I mean Parameters, not Input. See https://github.com/dnsimple/dnsimple-developer/pull/142/files#diff-12c45f6c13ebeebc73fec6c001707b85R95